cpm

```diff
@@ -120,7 +120,7 @@ fcount() {
 flist() {
     if [ -d "$1" ]; then
         cd "$1" || return
-        printf '%s\n' ./*
+        printf '%s\n' ./* | sed 's|./||g'
```

Comment:
err that's so far from what I was suggesting...

Reply:
yes, but it works